torch/onnx/_internal/fx/type_utils.py

```diff
@@ -66,6 +66,16 @@ def from_python_type_to_onnx_attribute_type(
     return _PYTHON_TYPE_TO_ONNX_ATTRIBUTE_TYPE.get(dtype)
 
 
+def is_torch_symbolic_type(t: Any) -> bool:
```

Comment:
```suggestion
def is_torch_symbolic_type(v: Any) -> bool:
```

nit: because it is instance of that type, not that type itself.